src/systems/filecoin_vm/runtime/impl/runtime.go

```diff
@@ -193,7 +194,7 @@ func (rt *VMContext) _createActor(codeID abi.ActorCodeID, address addr.Address)
 	// Create empty actor state.
 	actorState := &actor.ActorState_I{
 		CodeID_:     codeID,
-		State_:      actor.ActorSubstateCID(ipld.EmptyCID()),
+		State_:      actor.ActorSubstateCID(cid.Undef),
```

Comment:
I think we want this to be the CID of an empty state. The lotus code has something for this - could you replicate that pattern here?